pom.xml

```diff
@@ -58,6 +58,13 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.oceanbase</groupId>
+            <artifactId>obkv-table-client</artifactId>
+            <version>1.2.100-SNAPSHOT</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/obkv-table-client-1.2.100-SNAPSHOT-jar-with-dependencies.jar</systemPath>
```

Comment:
is this a tmp table-client jar?

Reply:
fixed

Reply:
This is the `obkv_params` branch of the repository at <https://github.com/oceanbase/obkv-table-client-java/tree/obkv_params>.